src/llmcompressor/modifiers/quantization/calibration.py

```diff
@@ -60,9 +66,17 @@ def initialize_observer(
         False,
         DynamicType.LOCAL,
     ):
+        observer_kwargs = quantization_args.observer_kwargs or {}
         observer = Observer.load_from_registry(
             quantization_args.observer,
             quantization_args=quantization_args,
+            maxshrink=observer_kwargs.get("maxshrink", DEFAULT_MAXSHRINK),
```

Comment:
Sorry, can we implement these defaults within the observe class implementations themselves? It's somewhat strange to pass `grid` into minmax observer